collect_app/src/test/java/org/odk/collect/android/activities/MainMenuActivityTest.java

```diff
@@ -0,0 +1,46 @@
+package org.odk.collect.android.activities;
+
+import android.content.Intent;
+import android.view.MenuItem;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.R;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.fakes.RoboMenuItem;
+import org.robolectric.shadows.ShadowEnvironment;
+
+import static android.os.Environment.MEDIA_MOUNTED;
+import static junit.framework.Assert.assertEquals;
+import static org.robolectric.Shadows.shadowOf;
+
+/**
+ * Unit test for checking {@link ScanQRCodeActivity} launched correctly from {@link MainMenuActivity}
```

Comment:
We probably don't need this comment. We'll most likely add tests to it so it'll get out of date fast!